script/set.sh

```diff
@@ -11,4 +11,6 @@
 # -u: treat unset variables as an error
 # -v: Print shell input lines as they are read
 # -x: Print command traces before executing command
+sudo sysctl vm.mmap_rnd_bits=28
```

Comment:
You was faster. I would also suggest it :-) It works.

Please guard it with:

```bash
if [ -n "$GITHUB_ACTIONS" ] && [ "$ALPAKA_CI_OS_NAME" = "Linux" ]; then
    sudo sysctl vm.mmap_rnd_bits=28
fi
```

An move it to `ci.sh`. `set.sh` is sourced many times and it's meaning is really executing the `set` command for the bash.